src/Libraries/Microsoft.Extensions.Telemetry/Microsoft.Extensions.Telemetry.json

```diff
@@ -15,10 +15,6 @@
           "Member": "System.Collections.Generic.IDictionary<string, System.Collections.Generic.ISet<string>> Microsoft.Extensions.Telemetry.Metering.EventCountersCollectorOptions.Counters { get; set; }",
           "Stage": "Stable"
         },
-        {
-          "Member": "System.TimeSpan Microsoft.Extensions.Telemetry.Metering.EventCountersCollectorOptions.EventListenerRecyclingInterval { get; set; }",
-          "Stage": "Stable"
```

Comment:
@geeknoid hopefully it's not too late to do such things

Reply:
Nope, not too late